pkgs/top-level/all-packages.nix

```diff
@@ -25609,7 +25609,10 @@ in
 
   chessx = libsForQt5.callPackage ../games/chessx { };
 
-  chiaki = libsForQt5.callPackage ../games/chiaki { };
+  chiaki = libsForQt5.callPackage ../games/chiaki {
+    libevdev = if stdenv.isDarwin then null else libevdev;
+    udev = if stdenv.isDarwin then null else udev;
+  };
```

Comment:
```suggestion
  chiaki = libsForQt5.callPackage ../games/chiaki { };
```

Reply:
won't this break darwin since `evdev` is only available on Linux?

Reply:
Nix has lazy evaluation, so unless you call the symbol it is not required.

Reply:
I see, other package using this scheme do this because they need to modify the configure stage for this.